test/helper/user.js

```diff
@@ -0,0 +1,94 @@
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+import User from 'common/User';
+import { createAndFindBooks } from 'test/helper/book';
+
+const defaultAccount = {
+  mail: 'dragonball@jump.com',
+  password: 'wakuwakusitekitazo',
+};
+
+function createUser(account = defaultAccount) {
+  const user = new UserModel(account);
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).